study.md

```diff
@@ -77,7 +86,10 @@ look up Ruby's array and string methods, then check the
     so that your result is `"c-o-f-f-e-e"`.
 
 ```ruby
-# your answer here
+a)str = "eeffoc"
+b) str.reverse
+c)str = str.split("")
+d)str = str.join("-")
```

Comment:
This will result in `"e-e-f-f-o-c"` since `str` wasn't reversed in place in step b (it should have been `str.reverse!`).